tests/robottelo/test_helpers.py

```diff
@@ -38,224 +45,224 @@ def test_return_none(self, ssh):
         When the versions.rb file is not present.
         """
         ssh.command = mock.MagicMock(return_value=FakeSSHResult([], 0))
-        self.assertEqual(get_server_version(), None)
+        assert get_server_version() is None
 
 
-class GetHostInfoTestCase(unittest2.TestCase):
+class TestGetHostInfo:
     """Tests for method ``get_host_credentials``."""
 
     @mock.patch('robottelo.helpers.ssh')
     def test_fedora_info(self, ssh):
         ssh.command = mock.MagicMock(
             return_value=FakeSSHResult(['Fedora release 20 (Heisenbug)'], 0)
         )
-        self.assertTupleEqual(get_host_info(), ('Fedora', 20, None))
+        assert get_host_info() == ('Fedora', 20, None)
 
     @mock.patch('robottelo.helpers.ssh')
     def test_rhel_info(self, ssh):
         ssh.command = mock.MagicMock(
             return_value=FakeSSHResult(['Red Hat Enterprise Linux Server release 7.1 (Maipo)'], 0)
         )
-        self.assertTupleEqual(get_host_info(), ('Red Hat Enterprise Linux Server', 7, 1))
+        assert get_host_info() == ('Red Hat Enterprise Linux Server', 7, 1)
 
     @mock.patch('robottelo.helpers.ssh')
     def test_cat_fail(self, ssh):
         ssh.command = mock.MagicMock(return_value=FakeSSHResult([], 1, 'stderr'))
-        with self.assertRaises(HostInfoError) as context:
+        with pytest.raises(HostInfoError) as context:
             get_host_info()
-        self.assertEqual(str(context.exception), 'Not able to cat /etc/redhat-release "stderr"')
+        assert context.match(r'.*Not able to cat /etc/redhat-release "stderr".*')
 
     @mock.patch('robottelo.helpers.ssh')
     def test_release_parse_fail(self, ssh):
         ssh.command = mock.MagicMock(return_value=FakeSSHResult([''], 0))
-        with self.assertRaises(HostInfoError) as context:
+        with pytest.raises(HostInfoError) as context:
             get_host_info()
-        message = str(context.exception)
-        self.assertEqual(message, 'Not able to parse release string ""')
+        assert context.match(r'.*Not able to parse release string "".*')
```

Comment:
this match should be able to be included in the raises context.
```python
with pytest.raises(HostInfoError, match=r'.*Not able to parse release string "".*'):
    get_host_info()
```
if you like that, there are other places this applies

Reply:
groovy, I like that syntax, will apply throughout.